utils/collect_validation_month_day_year_data.py

```diff
@@ -0,0 +1,77 @@
+import json
+from pprint import pprint
+import random
+from playwright.sync_api import sync_playwright
+from datetime import datetime
+
+
+def collect_validation_month_day_year_data(page):
```

Comment:
Check if both of methods should exist: the current one and `collect_test_run_month_day_year_data`

Reply:
If possible - make one method and use it in both for months and for years.